src/test/java/me/itzg/helpers/modrinth/ModrinthCommandTest.java

```diff
@@ -311,7 +311,7 @@ void handlesDatapacksSpecificVersion(boolean absoluteWorldDir, @TempDir Path tem
                     tempDir.resolve(worldDir).toString()
                     : worldDir,
                 "--game-version", "1.21.1",
-                "--loader", "datapack",
```

Comment:
Thanks, that must have been a typo originally, but masked by the "datapack:" prefix resolution.